macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -284,6 +286,20 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         return VPNUpsellUserDefaultsPersistor(keyValueStore: keyValueStore)
     }()
 
+    // MARK: - Home Page Continue Set Up Model
+
+    lazy var homePageSubscriptionCardVisibilityManager: HomePageSubscriptionCardVisibilityManaging = {
+        return HomePageSubscriptionCardVisibilityManager(subscriptionManager: subscriptionAuthV1toV2Bridge, persistor: homePageSubscriptionCardPersistor)
+    }()
+
+    lazy var homePageSubscriptionCardPersistor: HomePageSubscriptionCardPersisting = {
+        return HomePageSubscriptionCardPersistor(keyValueStore: keyValueStore)
+    }()
+
+    lazy var homePageContinueSetUpModelPersistor: HomePageContinueSetUpModelPersistor = {
+        return HomePageContinueSetUpModelPersistor(keyValueStore: UserDefaultsWrapper<Any>.sharedDefaults)
```

Comment:
Perhaps worth adding a comment that we’re sticking with UserDefaults here because the pre-existing code used it?